RELEASE.txt

```diff
@@ -40,6 +40,7 @@ IMPROVEMENTS
 - Verify that a user specified msvc script (via MSVC_USE_SCRIPT) exists and raise an
   exception immediately when the user specified msvc script does not exist.
 - Add cache-debug messages for push failures.
+- Add error message for failure to delete partial cache retrieval files.
```

Comment:
How about

"Add error message when SCons fails to delete files which will be retrieved from a CacheDir" ?

Reply:
"Currently this happens silently"

Reply:
OK how about this.
`Added error message to handle the case when SCons attempts to retrieve all the targets for a specified builder from the CacheDir, fails to do so, and then runs into an error when deleting the files which were retrieved. Previously if this happened there was no errors or warnings.`

Reply:
used your message in changes notes in 9dc6cbe